content/docs/setup/_index.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Setup and Upgrade
```

Comment:
Why remove Upgrade?

Reply:
All the other top level items are one word, so this one seems out of place. The way I look at it, at a high level it's all just part of setting up Istio. We don't mention Install at this level either. `Download, Install, and Upgrade` would be a better title if we did.